admin/src/app/foms/fom-add-edit/fom-add-edit.component.html

```diff
@@ -308,6 +395,15 @@ <h1 class="text-muted">{{isCreate ? 'Add New' : 'Edit'}} FOM {{isCreate ? '' : '
               formControlName="bctsMgrName"
               [appFormControl]="fg.get('bctsMgrName')"
             />
+            <input
+              *ngIf="!isInitialState"
+              type="text"
+              disabled
+              class="form-control"
+              id="mgr-name-id"
+              name="bctsMgrName"
+              [value]="fg.get('bctsMgrName').value"
+            />
```

Comment:
might be a good idea to reduce duplication to use ternary in places like this, for example, the two <input> can be merged into one by doing 
```html
<input
  type="text"
  class="form-control"
  id="mgr-name-id"
  name="bctsMgrName"
  formControlName="bctsMgrName"
  [appFormControl]="fg.get('bctsMgrName')"
  [attr.disabled]="!isInitialState ? true : null"
  [value]="isInitialState ? null : fg.get('bctsMgrName').value"
/>
```

Although this is fine for a hotfix, to the tech debt it goes!
<img src="https://media1.giphy.com/media/Ywe4I7SSzG8EBdbDIP/giphy-downsized-medium.gif" width="150"/>

Reply:
Thanks Craig!! In this case due to many interaction from Angular, I consider it for a bit of clarity over conciseness.